boards/renesas/rcar_salvator_xs_m3/board.yml

```diff
@@ -0,0 +1,5 @@
+board:
+  name: rcar_salvator_xs_m3
```

Comment:
Actually M3 is another name for r8a7796 SoC family. IIUC meaning of hwmv2, then generic board should be called `rcar_salvator_xs` and `m3` should be one of the SoC options.

Reply:
Fixed